velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -32,15 +32,21 @@ class SimpleAggregatesTest : public WindowTestBase {
             size, [](auto row) -> int32_t { return row % 10; }),
         makeFlatVector<int32_t>(
```

Comment:
You can remove "-> int32_t " in [](auto row) -> int32_t { return row % 7; }) in all the lambdas here.